lib/jekyll-import/version.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 module JekyllImport
-  VERSION = "0.14.0".freeze
+  VERSION = "0.14.0"
```

Comment:
We'll definitely want a version bump after this. :smiley:

Reply:
Yeah, let's do that outside this PR hahaha